Cargo.toml

```diff
@@ -45,6 +45,7 @@ libc = { version = "0.2", optional = true }
 socket2 = { version = "0.4", optional = true }
 
 [dev-dependencies]
+cargo-check-external-types = "0.1.2"
```

Comment:
Would it be possible to make the dependency only installed in the CI workflow? So that most local dev doesn't need to download the extra dep?